kafka/integration_test.go

```diff
@@ -2376,20 +2386,23 @@ func (its *IntegrationTestSuite) TestProducerConsumerHeaders() {
 
 func TestIntegration(t *testing.T) {
 	its := new(IntegrationTestSuite)
+	its.skipFlaky = true
```

Comment:
Please only set true by default if testconf.Semaphore == true. It's okay if these tests fail locally and in fact better since we are aware that something is flaky. This step will effectively remove the tests

Reply:
Why don't we remove this variable completely in that case?
Just use 

```
if  testconf.Semaphore {
                t.Skipf("Skipping TestConsumerCommitted since it is flaky[Does not run when tested with all the other integration tests]")
		return
```